internal/website/content/howto/secrets/open-keeper.md

```diff
@@ -0,0 +1,242 @@
+if err != nil {
+    return err
+}
+
+// Create a *secrets.Keeper.
+bucket, err := awskms.NewKeeper(kmsClient, "alias/master-key-test", nil)
+if err != nil {
+    return err
+}
+```
+
+[`awskms.NewKeeper`]: https://godoc.org/gocloud.dev/secrets/awskms#NewKeeper
+[AWS session]: https://docs.aws.amazon.com/sdk-for-go/api/aws/session/
+
+## Google Cloud Key Management Service
+
+The Go CDK can use keys from Google Cloud Platform's [Key Management
+Service][GCP KMS] (GCP KMS) to keep information secret. GCP KMS URLs are very
+similar to [key resource IDs][]:
+
+```go
+import (
+    "gocloud.dev/secrets"
+    _ "gocloud.dev/secrets/gcpkms"
+)
+
+// ...
+
+keeper, err := secrets.OpenKeeper(ctx,
+    "gcpkms://projects/MYPROJECT/" +
+    "locations/MYLOCATION/" +
+    "keyRings/MYKEYRING/" +
+    "cryptoKeys/MYKEY")
+if err != nil {
+    return err
+}
+```
+
+[GCP KMS]: https://cloud.google.com/kms/
+[key resource IDs]: https://cloud.google.com/kms/docs/object-hierarchy#key
+
+### Google Cloud Key Management Service Constructor
+
+The [`gcpkms.NewKeeper`][] constructor opens a GCP KMS key. You must first
+obtain [GCP credentials][] and then create a gRPC connection to GCP KMS.
+
+```go
+import (
+    "gocloud.dev/gcp"
+    "gocloud.dev/secrets/gcpkms"
+)
+
+// ...
+
+// Your GCP credentials.
+// See https://cloud.google.com/docs/authentication/production
+// for more info on alternatives.
+creds, err := gcp.DefaultCredentials(ctx)
+if err != nil {
+    return err
+}
+
+// Create a KMS client.
+kmsClient, err := gcpkms.Dial(ctx, gcp.CredentialsTokenSource(creds))
+if err != nil {
+    return err
+}
+
+// Create a *blob.Bucket.
+const keyID = gcpkms.KeyResourceID(
+    "MYPROJECT", "MYLOCATION", "MYKEYRING", "MYKEY")
+keeper, err := gcpkms.NewKeeper(client, keyID, nil)
+if err != nil {
+    return err
+}
+```
+
+[GCP credentials]: https://cloud.google.com/docs/authentication/production
+[`gcpkms.NewKeeper`]: https://godoc.org/gocloud.dev/secrets/gcpkms#NewKeeper
+
+## HashiCorp Vault
+
+The Go CDK can use the [transit secrets engine][] in [Vault][] to keep
+information secret. Vault URLs only specify the key ID. The Vault server
+endpoint and authentication token are specified using the environment
+variables `VAULT_SERVER_URL` and `VAULT_SERVER_TOKEN`, respectively.
+
+```go
+import (
+    "gocloud.dev/secrets"
+    _ "gocloud.dev/secrets/vault"
+)
+
+// ...
+
+keeper, err := secrets.OpenKeeper(ctx, "vault://my-key")
+if err != nil {
+    return err
+}
+```
+
+[Vault]: https://www.vaultproject.io/
+[transit secrets engine]: https://www.vaultproject.io/docs/secrets/transit/index.html
+
+### HashiCorp Vault Constructor
+
+The [`vault.NewKeeper`][] constructor opens a transit secrets engine key. You
+must first connect to your Vault instance.
+
+```go
+import (
+    "github.com/hashicorp/vault/api"
+    "gocloud.dev/secrets/vault"
+)
+
+// ...
+
+vaultClient, err := vault.Dial(ctx, &vault.Config{
+    Token: "<Client (Root) Token>",
```

Comment:
This is the first time you've used the angle-bracket syntax for a metavariable. Above you use, e.g `MYKEYRING`.